contract/contracts/Voting/Types.sol

```diff
@@ -9,13 +9,11 @@ library Types {
     }
 
     struct Candidate {
-        string name;
```

Comment:
did u delete the name because we can fetch it on db?

Reply:
Yes!